helperstuff/createdf_jes.py

```diff
@@ -339,6 +345,10 @@ def dataframes(year, doubleDiff, obs_reco, obs_reco_2nd):
     elif year == '2018':
         lumi_sig = 59.7
         lumi_bkg = 59.7
+    elif year == '2022':
+        lumi_sig = 7.9804
+        lumi_bkg = 7.9804
```

Comment:
shouldn't this be the full 2022 lumi? or alternative, shouldn't we have here the two periods separated?